### ISSUE TITLE
Tinker can still Blink-Dagger after Rearm despite taking damage?

### ISSUE THREAD

**Issue by Bogdanov89** [ValveSoftware/Dota2-Gameplay#15974]
### Ability name

Rearm

### Description

In patch 7.33 all Blink Daggers received this modification:
"Changed Blink from applying a 3-second damage downtime to a 3-second item mute duration."

However a friend of mine just had a 70 minute game where a Tinker kept blinking out of combat a bit too comfortably.

A lot of those cases were just where he was not hit for 3 seconds but some were a bit odd where it felt like a Warlock Golem's immolation aura or Venomancers poison or Vessel should have kept the Tinkers Blink muted but it did not.

Please investigate:
Match ID 7640893341
Date 17/3/2024

### Example Match ID (and possibly Timestamp)

Match ID 7640893341   Timestamp 34:40

**Comment by commentator24** [comment]
i didn't watch the replay but guardian greave / eul can dispel vessel etc
veno W poison sting does not interrupt blink

**Comment by kevlg** [comment]
That is because he was under the effect of eblade and reactive armor so no one was able to attack him to cancel his blink dagger.

**Comment by commentator24** [comment]
yup even if someone right click attack tinker or vessel, he will still can blink if Defense matrix still block 320 attack dmg.
Once you remove defense matrix dmg block then he can't blink.

please close this issue not a bug
